haystack/document_stores/elasticsearch.py

```diff
@@ -272,13 +272,27 @@ def get_documents_by_id(
         headers: Optional[Dict[str, str]] = None,
     ) -> List[Document]:
         """
-        Fetch documents by specifying a list of text id strings. Be aware that passing a large number of ids might lead
-        to performance issues. Note that Elasticsearch limits the number of results to 10,000 documents by default.
+        Fetch documents by specifying a list of text id strings.
+
+        :param ids: list of document IDs. Be aware that passing a large number of ids might lead to performance issues.
+        :param index: Elasticsearch index where the meta values should be searched. If not supplied,
```

Comment:
By meta values, are you referring to the metadata fields that we use for filtering? Where are they set?

Reply:
It was a wrong copy-paste.